great_expectations/expectations/metrics/query_metrics/query_multiple_columns.py

```diff
@@ -81,7 +81,7 @@ def _sqlalchemy(
         engine: sqlalchemy_engine_Engine = execution_engine.engine
         result: List[sqlalchemy_engine_Row] = engine.execute(sa.text(query)).fetchall()
 
-        return [dict(element) for element in result]
+        return [element._asdict() for element in result]
```

Comment:
❤️ -- great thing (I thought had already done this for Query metrics -- but maybe it got reversed.  We need to be careful here.  If I recall correctly, I had tried this, but it did not work for one of the users (who was on a lower version of SQL in their environment).  Happy to discuss.

Reply:
Ah that might have been true until relatively recently. We bumped our minimum required `sqlalchemy` version to `1.4.0` which contains the `_asdict()` method. 

https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Row._asdict